### ISSUE TITLE
[BUG] Pipeline breaks when '--skip_qc' is chosen

### ISSUE THREAD

**Issue by wanyuac** [gbouras13/hybracter#51]
**Describe the bug**
Hybracter v0.5.0 failed to provide input reads for flye when option '--skip_qc' is specified.

**To Reproduce**
Command:
```
hybracter hybrid-single -l reads/nanopore/sample1.fastq -1 reads/illumina/sample1_1.fastq.gz -2 reads/illumina/sample1_2.fastq.gz -s sample1 -c 5000000 -o sample1 -t 16 --skip_qc --medakaModel r941_min_fast_g507 --flyeModel '--nano-raw' --logic best --use-conda
```

Error message:
```
[Fri Jan 19 06:15:51 2024]
Error in rule assemble:
    jobid: 6
    input: sample1/processing/qc/sample1_filt_trim.fastq.gz
    output: sample1/processing/assemblies/sample1/assembly.fasta, sample1/processing/assemblies/sample1/assembly_info.txt, sample1/versions/sample1/flye.version, sample1/supplementary_results/flye_individual_summaries/sample1_assembly_info.txt
    log: sample1/stderr/assemble/sample1.log (check log file(s) for error details)
    conda-env: /home/ywan1/.conda/envs/hybracter/lib/python3.12/site-packages/hybracter/workflow/conda/c1d130cca38fe643cfb099e9206899d6_
    shell:
        
        flye --nano-raw sample1/processing/qc/sample1_filt_trim.fastq.gz -t 16  --out-dir sample1/processing/assemblies/sample1 2> sample1/stderr/assemble/sample1.log
        flye --version > sample1/versions/sample1/flye.version
        cp sample1/processing/assemblies/sample1/assembly_info.txt sample1/supplementary_results/flye_individual_summaries/sample1_assembly_info.txt
        
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)
Logfile sample1/stderr/assemble/sample1.log:
=======================================================
[2024-01-19 06:15:51] INFO: Starting Flye 2.9.3-b1797
[2024-01-19 06:15:51] INFO: >>>STAGE: configure
[2024-01-19 06:15:51] INFO: Configuring run
[2024-01-19 06:15:51] ERROR: Not a gzipped file (b'@b')
[2024-01-19 06:15:51] ERROR: Pipeline aborted
=======================================================

Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
```

**Expected behavior**
Pipelines runs without QC steps.

**Desktop (please complete the following information):**
- OS: Linux — Ubuntu 22.04.3 LTS
- Conda v23.11.0

**Additional context**
Have attached the log file for information.
[2024-01-19T061548.825444.snakemake.log](https://github.com/gbouras13/hybracter/files/13985179/2024-01-19T061548.825444.snakemake.log)

**Comment by gbouras13** [comment]
Hi @wanyuac ,

The issue is your long reads are not gzipped.

This is certainly a bug that I will fix soon. For now if you gzip the Fastqs and rerun it should work.

George

**Comment by wanyuac** [comment]
I see, thank you for the suggestion. I preprocessed long reads for quality so reads were left uncompressed. Noting the use of porechop-ABI in the pipeline, does this tool can handle reads from both old barcoding kits (e.g., the R9 rapid barcoding kit SQK-RBK004) and new ones (namely, those of V14 chemistry for R10 flow cells, such as SQK-RBK114.24)?

**Comment by gbouras13** [comment]
Hi @wanyuac ,

Just getting to this now - the bug fix will be part of v0.7.0.

Regarding your question, I haven't rigorously tested on all kits, but I have run R9.4.1 96x rapid kit (RBK110-96) and R10.4.1 (RBK114-96) data through hybracter with no issues. Porechop-ABI has support for kits going as far back as Porechop does, so it shouldn't be an issue, but of course if you do some more investigation please let me know. 

Ultimately, I just include the QC for convenience - a lot of users will want to do their own QC prior I'm sure, so thanks for spotting this important bug!

George
